src/main/scala/io/iohk/ethereum/domain/Blockchain.scala

```diff
@@ -307,16 +313,16 @@ class BlockchainImpl(
       rootHash: ByteString,
       position: BigInt,
       ethCompatibleStorage: Boolean
-  ): Option[(BigInt, Seq[MptNode])] = {
+  ): (BigInt, Seq[MptNode]) = {
+    val defaultValue = BigInt(0)
```

Comment:
.. and default value together with empty seq can become an `EmptyProof` like in case above

Reply:
@dmitry-worker so I made a change here, and replaced a tuple with a `StorageValueProof` class from EthProofService, but now I see that this way we are coupling 2 different modules and putting specifics of implementation into a Blockchain trait. What do you think?

Reply:
I like the way it looks now, perfectly readable. As for the JsonRPC classes imported into a blockchain - it's a good point and I would ask @dzajkowski - does he prefer to move those guys from JsonRPC to the blockchain? Or maybe to create an internal image for these classes. This can be done afterwards, in a separate PR.